apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -34,6 +35,8 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
     state => state.lab2Project.projectSources?.source
   ) as MultiFileSource | undefined;
   const files = source?.files || {};
+  // The resource panel includes settings.
+  const showSettings = !isUsingResourcePanel(appName);
```

Comment:
In `RightButtons`, you may want to do a similar check as the settings button is included in widget view.

Reply:
good catch!